scripts/zones/Arrapago_Reef/MobIDs.lua

```diff
@@ -1,4 +1,5 @@
 LIL_APKALLU             = 16998871;
-VELIONIS                = 16998872;
-ZAREEHKL_THE_JUBILANT   = 16998873;
+MEDUSA                  = 16998862;
```

Comment:
lets not move the id's out of order when we add one.

Reply:
i thought while here may as well put in alpha order, should help your ocd lol